modules/setting-up-an-openshift-login.adoc

```diff
@@ -0,0 +1,90 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/getting-started--with-openshift-gitops.adoc
+
+[id="setting-up-an-openshift-login_{context}"]
+= Setting up an OpenShift login
+
+== Creating a new client in Keycloak
+
+To set up an OpenShift login, you must create a new client in Keycloak:
+
+. Log in to your Keycloak server.
+. Navigate to *myrealm* -> *Clients* ->  and then select *Create*:
+.. For *Client ID*, type *argocd*.
+.. For *Client Protocol*, select *openid-connect*.
+.. For *Root URL*, select the Argo CD route URL.
+. Configure the client:
+.. For *Access Type*, select *confidential*.
+.. For *Valid Redirect URLs*, select the callback url for your Argo CD hostname. Your callback should be `https://{hostname}/auth/callback`.
+.. Set the *Base URL* to */applications*.
+.. Save your work and then you get the *Credentials* tab. Go to this tab.
+.. For *Client Authenticator*, select *Client ID and Secret* and copy the *argocd-secret* so you can use it for your Argo CD configuration.
+
+== Configuring Argo CD OpenID Connect (OIDC)
+
+After you create a new client in Keycloak and generate an *argocd-secret* in the Argo CD secret, you get a client secret. To configure Argo CD OIDC, store this client secret with the following instructions:
+
+. Encode the client secret in base64:
++
+[source,terminal]
+----
+$ echo -n '83083958-8ec6-47b0-a411-a8c55381fbd2' | base64
+----
+
+. Add the base64 value to a new key called `oidc.keycloak.clientSecret` and edit the secret:
```

Comment:
From this point on, you've completely lost me. The descriptions of the steps don't appear to correspond to the commands that you're using, and I am extremely confused what of the YAML you're showing as example _is actually there_ and what the user needs to customize. The YAML format itself looks slightly off - I've never seen a YAML example start with a "$." Please review the rest of the module and try to be clearer. Use the annotation format from some of the other YAML examples to explain what's going on.